### ISSUE TITLE
armel/armhf/mips*: L7 fuzzing test: Makefile:963: recipe for target 'rewrite_l7fuzzing' failed

### ISSUE THREAD

**Issue by fklassen** [appneta/tcpreplay#364]
See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=861539

Related to new Fuzz feature, fails tests on some platforms.

```
echo Making test in ./test
Making test in ./test
cd ./test && make test
make[1]: Entering directory `/home/user/src/tcpreplay-4.2.5-beta1/test'
NOTICE: Tests must be run as root
Sending traffic on 'eth0' and 'eth0'
[tcpprep] Auto/Router mode test: 		OK
[tcpprep] Auto/Bridge mode test: 		OK
[tcpprep] Auto/Client mode test: 		OK
[tcpprep] Auto/Server mode test: 		OK
[tcpprep] Auto/First mode test: 		OK
[tcpprep] CIDR mode test: 			OK
[tcpprep] Regex mode test: 			OK
[tcpprep] Port mode test: 			OK
[tcpprep] MAC mode test: 			OK
[tcpprep] Comment mode test: 			OK
[tcpprep] Print info mode test: 		OK
[tcpprep] Print comment mode test: 		OK
[tcpprep] Config mode test: 			OK
[tcpprep] MAC reverse mode test: 		OK
[tcpprep] CIDR reverse mode test: 		OK
[tcpprep] Regex reverse mode test: 		OK
[tcpreplay] Basic test: 			OK
[tcpreplay] Cache test: 			OK
[tcpreplay] Packets/sec test: 			OK
[tcpreplay] Mbps test: 				OK
[tcpreplay] Topspeed test: 			OK
[tcpreplay] Config file/VLAN add test: 		OK
[tcpreplay] Multiplier test: 			OK
[tcpreplay] Packets/sec Multiplier test: 	OK
[tcpreplay] Precache test: 			OK
[tcpreplay] Statistics test: 			OK
[tcpreplay] Dual file test: 			OK
[tcpreplay] Maximum sleep test: 		OK
[tcprewrite] Portmap test: 			OK
[tcprewrite] Endpoint test: 			OK
[tcprewrite] Pseudo NAT test: 			OK
[tcprewrite] Truncate test: 			OK
[tcprewrite] Pad test: 				OK
[tcprewrite] Seed IP test: 			OK
[tcprewrite] Src/Dst MAC test: 			OK
[tcprewrite] Layer2 test: 			OK
[tcprewrite] Config/VLAN Add test: 		OK
[tcprewrite] Skip bcast test: 			OK
[tcprewrite] DLT User test: 			OK
[tcprewrite] DLT Cisco HDLC test: 		OK
[tcprewrite] VLAN Delete test: 			OK
[tcprewrite] Remove EFCS: 			OK
[tcprewrite] Force TTL: 			OK
[tcprewrite] Increase TTL: 			OK
[tcprewrite] Reduce TTL: 			OK
[tcprewrite] TOS test: 				OK
[tcprewrite] MTU Truncate test: 		OK
[tcprewrite] Substitute Src/Dst MAC test: 	OK
[tcprewrite] Seeded MAC test: 			OK
[tcprewrite] Seeded Keep MAC test: 		OK
[tcprewrite] L7 fuzzing test: make[1]: *** [rewrite_l7fuzzing] Error 2
make[1]: Target `test' not remade because of errors.
make[1]: Leaving directory `/home/user/src/tcpreplay-4.2.5-beta1/test'
make: *** [test] Error 2
```

**Comment by fklassen** [comment]
Sayeth Christoph Biedl:

```
On armel, FORCE_ALIGN
is defined, hence tcpedit_dlt_l3data_copy uses a different code path;
and the returned pointer is not suitable for the pointer arithmetics
fuzz_get_datalen does. So datalen gets ridiculously huge values, and
eventually fuzzing bails out without doing anything.
```

**Comment by fklassen** [comment]
Compiled `armel` with 3.4.3 source from Debian source package. It appears that tests were not even close to passing on that platform:

```
echo Making test in ./test
Making test in ./test
cd ./test && make test
make[1]: Entering directory `/home/user/src/tcpreplay-debian/tcpreplay-3.4.3/test'
NOTICE: Tests must be run as root
Sending traffic on 'eth0' and 'eth0'
[tcpprep] Auto/Router mode test: 		OK
[tcpprep] Auto/Bridge mode test: 		OK
[tcpprep] Auto/Client mode test: 		OK
[tcpprep] Auto/Server mode test: 		OK
[tcpprep] Auto/First mode test: 		OK
[tcpprep] CIDR mode test: 			OK
[tcpprep] Regex mode test: 			OK
[tcpprep] Port mode test: 			OK
[tcpprep] MAC mode test: 			OK
[tcpprep] Comment mode test: 			OK
[tcpprep] Print info mode test: 		OK
[tcpprep] Print comment mode test: 		OK
[tcpprep] Config mode test: 			OK
[tcpprep] MAC reverse mode test: 		OK
[tcpprep] CIDR reverse mode test: 		OK
[tcpprep] Regex reverse mode test: 		OK
[tcpreplay] Basic test: make[1]: *** [replay_basic] Error 255
[tcpreplay] Cache test: make[1]: *** [replay_cache] Error 255
[tcpreplay] Packets/sec test: make[1]: *** [replay_pps] Error 255
[tcpreplay] Mbps test: make[1]: *** [replay_rate] Error 255
[tcpreplay] Topspeed test: make[1]: *** [replay_top] Error 255
[tcpreplay] Config file/VLAN add test: make[1]: *** [replay_config] Error 255
[tcpreplay] Multiplier test: make[1]: *** [replay_multi] Error 255
[tcpreplay] Packets/sec Multiplier test: make[1]: *** [replay_pps_multi] Error 255
[tcprewrite] Portmap test: make[1]: *** [rewrite_portmap] Error 2
[tcprewrite] Endpoint test: make[1]: *** [rewrite_endpoint] Error 2
[tcprewrite] Pseudo NAT test: make[1]: *** [rewrite_pnat] Error 2
[tcprewrite] Truncate test: make[1]: *** [rewrite_trunc] Error 2
[tcprewrite] Pad test: make[1]: *** [rewrite_pad] Error 2
[tcprewrite] Seed IP test: make[1]: *** [rewrite_seed] Error 2
[tcprewrite] Src/Dst MAC test: make[1]: *** [rewrite_mac] Error 2
[tcprewrite] Layer2 test: make[1]: *** [rewrite_layer2] Error 2
[tcprewrite] Config/VLAN Add test: make[1]: *** [rewrite_config] Error 2
[tcprewrite] Skip bcast test: make[1]: *** [rewrite_skip] Error 2
[tcprewrite] DLT User test: make[1]: *** [rewrite_dltuser] Error 2
[tcprewrite] DLT Cisco HDLC test: make[1]: *** [rewrite_dlthdlc] Error 2
[tcprewrite] VLAN Delete test: 			OK
[tcprewrite] Remove EFCS: make[1]: *** [rewrite_efcs] Error 2
[tcprewrite] Force TTL: make[1]: *** [rewrite_1ttl] Error 2
[tcprewrite] Increase TTL: make[1]: *** [rewrite_2ttl] Error 2
[tcprewrite] Reduce TTL: make[1]: *** [rewrite_3ttl] Error 2
[tcprewrite] TOS test: make[1]: *** [rewrite_tos] Error 2
[tcprewrite] MTU Truncate test: make[1]: *** [rewrite_mtutrunc] Error 2
make[1]: Target `test' not remade because of errors.
make[1]: Leaving directory `/home/user/src/tcpreplay-debian/tcpreplay-3.4.3/test'
make: *** [test] Error 2
```

**Comment by fklassen** [comment]
Fixed in #373
